src/Polysemy/Internal.hs

```diff
@@ -103,7 +103,8 @@ import Polysemy.Internal.Union
 --         catching = do
 --           modify (++"-catch")
 --           get
---     catch @String throwing (\ _ -> catching)
+--     -- Without `-fplugin=Polysemy.Plugin` you need `catch @String`
```

Comment:
Nit:
Since these are Haskell comments, maybe the backticks should be `@`s instead? Not sure if haddock even checks nested comments.

Reply:
Haddock doesn't check these AFAIK